From hjk@linutronix.de Thu Feb  7 05:42:50 2008
From: Hans-Jürgen Koch <hjk@linutronix.de>
Date: Thu, 7 Feb 2008 13:38:05 +0100
Subject: UIO: Make UIO available on ARM architecture
To: linux-kernel@vger.kernel.org
Cc: linux-arm-kernel@lists.arm.linux.org.uk, Greg KH <greg@kroah.com>
Message-ID: <20080207133805.13f092ff@dilbert.local>


Subject: arch/arm/Kconfig: Make UIO available on ARM architecture

Source drivers/uio/Kconfig to make UIO available in menuconfig if ARCH=arm.
I already posted this a few months ago, but it got lost somehow.

Signed-off-by: Hans J Koch <hjk@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 arch/arm/Kconfig |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1083,6 +1083,8 @@ source "kernel/power/Kconfig"
 config ARCH_SUSPEND_POSSIBLE
 	def_bool y
 
+source "drivers/uio/Kconfig"
+
 endmenu
 
 source "net/Kconfig"
